src/contrib/mlir/compiler/compiler.cpp

```diff
@@ -637,25 +637,39 @@ namespace ngraph
 template <typename Op>
 mlir::Operation* MLIRCompiler::createGenericOp(const ngraph::Node* ngNode)
 {
-    std::vector<mlir::Value*> argValues;
-    std::vector<mlir::Type> resTypes;
-    for (auto& arg : ngNode->get_arguments())
+    std::vector<mlir::Value*> arg_values;
```

Comment:
nit: we are following MIR naming convention in this file so we should use `lowerCamelCase` case for all the variable names, params, function names, etc. You can check the naming convention used in MLIR files by looking at the NOTE at the beginning of the file.